tests/template/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -107,7 +107,7 @@ async function f() {
 b()\`\`;
 
 // \\"ClassExpression\\"
-(class {})\`\`;
+(class {}\`\`);
```

Comment:
Both are valid (well, they both throw at runtime, but parse the same), so it doesn't really matter which one we output. Since this is such an edge case, it's probably never going to be used in practice and whichever we output is fine.